content/zh/docs/tutorials/kubernetes-basics/create-cluster/_index.md

```diff
@@ -2,3 +2,9 @@
 title: 创建集群
 weight: 10
 ---
+
+<!--
+Learn about Kubernetes {{< glossary_tooltip text="cluster" term_id="cluster" length="all" >}} and create a simple cluster using Minikube.
+-->
+了解 Kubernetes {{<glossary_tooltip text="集群" term_id="cluster" length="all" >}} 并使用 Minikube
```

Comment:
when a `glossary_tooltip` will be rendered into Chinese text, we treat it as Chinese text, so the extra space is not needed.
```suggestion
了解 Kubernetes {{<glossary_tooltip text="集群" term_id="cluster" length="all" >}}并使用 Minikube
```